internal/usage/usage.go

```diff
@@ -174,7 +174,7 @@ dbName and collection are required only for built-in roles.`
 	SearchFields                              = "Static field specifications."
 	Key                                       = "Field to be indexed and the type of index in the following format: field:type."
 	Sparse                                    = "Flag that creates a sparse index. To learn more, see https://dochub.mongodb.org/core/index-sparse-manual."
-	ClusterName                               = "Name of the cluster."
+	ClusterName                               = "Name of the cluster. To learn more, see https://dochub.mongodb.org/core/create-cluster-api."
```

Comment:
I've heard before and maybe this is a good opportunity to confirm, if we end it with a period(.) when clicking on the terminal it'll include the dot in the url and sometimes produce weird thing sin browsers, can you confirm this works if you try to click from the termial?

Reply:
It works correctly using ITerm and zshell 
![Screenshot 2024-05-02 at 10 17 15](https://github.com/mongodb/mongodb-atlas-cli/assets/5663078/1538822a-d133-47cb-bcb2-db6f19e2ba6a)

Reply:
the terminal app of the mac does not work for some reason on my machine 😭